test/compilable/traits.d

```diff
@@ -54,3 +54,8 @@ static assert(__traits(isModule, b));
 // This is supposed to work even though we haven't directly imported imports.pkgmodule.
 static assert(__traits(isModule, imports.pkgmodule));
 static assert(!__traits(isModule, MyStruct));
+
+version(Windows)
+    static assert(__traits(getLocation, MyStruct)[0] == "compilable\\traits.d");
```

Comment:
I recommend using backticks for the string literal to avoid the need for escaping `\`.